python/dolfinx/plot.py

```diff
@@ -64,7 +64,7 @@ def vtk_mesh(msh: mesh.Mesh, dim: typing.Optional[int] = None, entities=None):
         vtk_topology = _cpp.io.extract_vtk_connectivity(msh.geometry.dofmap, cell_type)[entities]
         num_nodes_per_cell = vtk_topology.shape[1]
     else:
-        # NOTE: This linearizes higher order geometries
+        # NOTE: This linearizes higher order geometries FIXME no longer true
```

Comment:
Fix comment.

Reply:
I'm not sure what the best thing is to do here. `entities_to_geometry` used to linearise geometry. This is no longer the case, so I doubt this code will work now if high-order geometry is used.

There are two options:
1) We could just get the DOFs associated with vertices from the output of `entities_to_geometry`
2) We generalise this function to work with high-order geometry

Option 2 is obviously better, but would need more work. What are your thoughts @jorgensd?

Reply:
@jpdean do you have a change coming on this?

Reply:
I can have a look at this if Joe doesn't have time.

Reply:
> I can have a look at this if Joe doesn't have time.

Great. Let's not leave this PR hanging around.

Reply:
Sorry, I haven't had chance to look at this yet!

Reply:
Plotting logic simplified alot.